godel.rosinstall

```diff
@@ -6,3 +6,5 @@
     version: indigo-devel}
 - git: {local-name: motoman, uri: 'https://github.com/ros-industrial/motoman.git',
     version: indigo-devel}
+- git: {local-name: abb, uri: 'https://github.com/ros-industrial/abb.git',
+    version: indigo-devel}
```

Comment:
`ABB` is available in Indigo ([ros_indigo_default.html?q=abb](http://repositories.ros.org/status_page/ros_indigo_default.html?q=abb), v1.2.0, same as in `indigo-devel`), did `rosdep` not resolve it?